client/src/components/__tests__/Calendar.spec.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+import moment from 'moment';
+
+import Calendar from '../Calendar';
+import { wrap } from 'module';
+describe('Calendar', () => {
+
+
```

Comment:
lots of empty space

Reply:
Thanks! I made a commit to properly space out that file